lib/ruby_sucks.rb

```diff
@@ -0,0 +1,92 @@
+# Challenge 1 - Calculator
+# Create a simple calculator that first asks the user what method they would
+# like to use (addition, subtraction, multiplication, division) and then asks
+# the user for two numbers, returning the result of the method with the two
+# numbers. Here is a sample prompt:
+
+# puts  'What calculation would you like to do? (add, sub, mult, div)'
+# calculation = gets.chomp.strip
+
+#     puts "What is the first number you want to '#{calculation}'?"
+#     num1 = gets.chomp.strip
+
+#     puts "What is the second number you want to '#{calculation}'?"
+#     num2 = gets.chomp.strip
+
+#     if calculation == 'add'
+#       results = num1.to_i + num2.to_i
+#     elsif calculation == 'sub'
+#       results = num1.to_i - num2.to_i
+#     elsif calculation == 'mult'
+#       results = num1.to_i * num2.to_i
+#     elsif calculation == 'div'
+#       results = num1.to_i / num2.to_i
+#     else
+# end
+
+
+# puts "Your result is '#{results}'"
+
+
+
+# Challenge 2 - Reverese A String
+
+puts  'Enter a string:'
+reverse = gets.chomp.strip
+
+arr = reverse.chars
+# char_count = reverse.chars.count
+
+for i in (0..(arr.length-1)/2)
+  arr[arr.length-1-i], arr[i] = arr[i], arr[arr.length-1-i]
+end
+
+puts arr.inspect
```

Comment:
instead of arr.inspect use arr.join to convert it back in to a string and display it